demo-react/src/components/ComingSoon.tsx

```diff
@@ -0,0 +1,173 @@
+export const ComingSoon = () => {
```

Comment:
OOI why use named exports here and not `export default`? What benefit does this give?

Reply:
My arguments are usually
1. Less typing, I can do `export * from './Component'` in root `index.ts` instead of `import { default as ... } ...`
2. Always the same name everywhere we import it, not giving freedom to rename to whatever the devs like
3. Some argue it's better for tree shaking as well (https://stackoverflow.com/a/68665805
4. Better message in error stack, it lists the name of the component (rarely using it tho)

For this case, I don't really care tho, none of these really matter for such a small and simple code base. We can use defaults 🤷 😆

Reply:
Just to clarify, you mean consistency across repos/project, meaning I should change those to default or do you mean consistency only inside those `demo-react` folder, and I can leave those for now? 😅

Reply:
no need to change

Reply:
consistent within a codebase :)